pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -225,6 +227,11 @@ func NewKubeGenericRuntimeManager(
 		}
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.DockerAuthPlugins) && dockerAuthPluginFile != "" {
+		external.RegisterExternalProviders(dockerAuthPluginFile)
```

Comment:
Should we be doing something here if register returns an error? Logging at a minimum.

Reply:
yeah we should